homeassistant/components/sensor/owfs.py

```diff
@@ -0,0 +1,71 @@
+"""
```

Comment:
We're all cool now, you can store platforms inside the component as: `owfs/sensor.py`